### PR TITLE
Added 1tb as the move limitation

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
1Tb size limit for ODB Move

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/move-onedrive-between-geo-locations.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can schedule OneDrive site moves in advance (described later in this article
 
 - You can schedule up to 4,000 moves at a time.
 - As the moves begin, you can schedule more, with a maximum of 4,000 pending moves in the queue and any given time.
-- We recommend not scheduling more than 4,000 moves per month.
+- The maximum size of a OneDrive that can be moved is 1 terabyte (1 TB)
```
Comment (JohanFreelancer9) [inline Enterprise/move-onedrive-between-geo-locations.md]:
```suggestion
- The maximum size of a OneDrive that can be moved is 1 terabyte (1 TB).
```

**Comment by mypil** [comment]
@AdwoodMicrosoft  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy/edited and there are just some minor edits but we got no response from AdwoodMicrosoft to merge the commits so we leave it to your discretion whether or not to merge this PR. 

Thank you.

cc: @kenwith

**Pull request merged**